app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -58,17 +65,40 @@ public Media createFromParcel(Parcel parcel) {
     protected String licenseUrl;
     protected String creator;
     protected ArrayList<String> categories; // as loaded at runtime?
+    /**
+     * Depicts is a feature part of Structured data. Multiple Depictions can be added for an image just like categories.
+     * However unlike categories depictions is multi-lingual
+     * Some expected properties include
+     * location, depicted in background, clothing, formatting (e.g. monochrome images or daguerreotypes)
+     * season/date/time, photographer or artist
+     */
+    protected ArrayList<Map<String, String>> depictionList;
+    protected ArrayList<String> depictions;
     protected boolean requestedDeletion;
     private Map<String, String> descriptions; // multilingual descriptions as loaded
+    /**
+     * This hasmap stores the list of multilingual captions, where
+     * key of the HashMap is the language and value is the caption in the corresponding language
+     * Ex: key = "en", value: "<description in short in English>"
+     *     key = "de" , value: "<description in german>"
+     */
+    private HashMap<String, String> captions;
     private HashMap<String, Object> tags = new HashMap<>();
     private @Nullable LatLng coordinates;
 
+    /**
+     * Attribute of standard page object as returned by the MediaWiki API.
```

Comment:
Please give an example value for an actual Commons file (give both URL and pageId)

Reply:
@nicolas-raoul this variable pageId was not added by me. Also I see that there are no usages of the pageId on the code. Shall i remove this variable